pyanaconda/display.py

```diff
@@ -201,14 +224,34 @@ def wl_preexec():
         # to set GUI subprocess SIGINT handler
         signal.signal(signal.SIGINT, signal.SIG_IGN)
 
+    # lets compile arguments for the run-in-new-session script
     argv = ["run-in-new-session.py",
             "--user", "root",
             "--service", "anaconda",
-            "--vt", "6",
             "--session-type", "wayland",
-            "--session-class", "user",
-            "gnome-kiosk", "--sm-disable", "--wayland", "--no-x11",
-            "--wayland-display", constants.WAYLAND_SOCKET_NAME]
+            "--session-class", "user"]
+
+    if headless:
+        # headless (remote connection) - stay on VT1 where connection info is
+        argv.extend(["--vt", "1"])
+    else:
+        # local display - switch to VT6 & show GUI there
+        argv.extend(["--vt", "6"])
+
+    # add the generic GNOME Kiosk invocation
+    argv.extend(["gnome-kiosk", "--sm-disable",
+                 "--wayland", "--no-x11",
+                 "--wayland-display", constants.WAYLAND_SOCKET_NAME])
+
+    # remote access needs gnome-kiosk to start in headless mode &
+    # configure a virtual monitor
+    if headless:
+        # check virtual monitor  resolution has been set
+        if headless_resolution is None:
+            # use default value
+            headless_resolution = "1280x1024"
+        # FIXME: use inst.resolution for the virtual monitor
```

Comment:
I believe `options.runres` is the right option and it should be working at the moment?

Reply:
`1280x1024` is huge! Shouldn't this be `1024x768`?